tests/conftest.py

```diff
@@ -32,19 +40,19 @@ def _init_console(locale_dir=sphinx.locale._LOCALE_DIR, catalog='sphinx'):
 
 
 @pytest.fixture(scope='session')
-def rootdir():
+def rootdir() -> Path:
     return Path(__file__).parent.resolve() / 'roots'
 
 
-def pytest_report_header(config):
+def pytest_report_header(config: pytest.Config) -> str:
     header = f"libraries: Sphinx-{sphinx.__display_version__}, docutils-{docutils.__version__}"
     if hasattr(config, '_tmp_path_factory'):
         header += f"\nbase tmp_path: {config._tmp_path_factory.getbasetemp()}"
     return header
 
 
 @pytest.fixture(autouse=True)
-def _cleanup_docutils():
+def _cleanup_docutils() -> Generator[None, None, None]:
```

Comment:
you can also use `Iterable[None]` i think, if that's cleaner?

Reply:
`Iterator` would be better IMO but fixtures like that are typed as generators for Pytest, so I think it's better to keep that. Also, I feel that an iterator should be used when you expect an iterator (and would do something like for x in ...) but here it's closer to a context.